.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
[OPTIONAL] well done for adding the test.md file to the .gitignore file. You should also consider adding the node modules in subsequent projects since it is too bulky to push into github repository